distribution/docs/src/main/resources/content/_configuring/admin-theming-configurations.adoc

```diff
@@ -0,0 +1,10 @@
+:title: Admin Theming Configurations
```

Comment:
Can we make this "Configuring UI themes"? to be parallel with the other section titles?

Reply:
fixed